docs/sts/keycloak.md

```diff
@@ -34,11 +32,10 @@ mc admin service restart myminio
 ```
 
 ## 4. Using WebIdentiy API
-
-Client ID and Client Secret can be found by clicking any of the clients listed [here](http://localhost:8080/auth/admin/master/console/#/realms/demo/clients). If you have followed the above steps docs, the default Client ID will be `account` and Client Secret can be found under `Credentials` tab.
+Client ID can be found by clicking any of the clients listed [here](http://localhost:8080/auth/admin/master/console/#/realms/demo/clients). If you have followed the above steps docs, the default Client ID will be `account`.
 
 ```
-$ go run web-identity.go -cid account -csec e61cb282-745b-4113-bece-29b921c735f0 -auth-ep http://localhost:8080/auth/realms/demo/protocol/openid-connect/auth -token-ep http://localhost:8080/auth/realms/demo/protocol/openid-connect/token -port 8888
+$ go run web-identity.go -cid account -config-url "http://localhost:8080/auth/realms/demo/.well-known/openid-configuration" -port 8888
```

Comment:
Running it with keycloak

`go run web-identity.go -cid account -config-ep "http://localhost:8081/auth/realms/demo/.well-known/openid-configuration" -port 8888`

throws following error 
```
oauth2: cannot fetch token: 400 Bad Request
Response: {"error":"unauthorized_client","error_description":"Client secret not provided in request"}
```

Reply:
`-config-url` should be `-config-ep`

Reply:
Added back the csec, since implicit flow is not possible without full browser support. It cannot be done using the command line by just writing Go code. Should work fine now.